release-notes.txt

```diff
@@ -1,3 +1,390 @@
+       LTO
+#5848: arduino: Race condition in sys/arduino/Makefile.include
+#6018: nRF52: gnrc 6lowpan ble memory leak
+#6437: periph/spi: Leftovers from SPI rework in #4780
+#6567: periph/spi: Switching between CPOL=0,1 problems on Kinetis with software
+       CS
+#6650: GPIO Port shift in cc2538 is unconsistent
+#6874: SAMD21: possible CMSIS bug
+#7020: isr_rfcoreerrors while pinging between CC2538DKs
+#7667: sam0 flashpage_write issues
+#7846: stm32f1: I2C read functions return bogus values with unconnected devices
+#8028: diskio: failed assertion in send_cmd() on lpc2387
+#8045: stm32/periph/uart: extra byte transmitted on first transmission
+#8436: Kinetis PhyNode: failed to flash binary > 256K
+#8444: tests/thread_priority_inversion hangup
+#9619: ATmega platform issues.
+#9882: sys/tsrb is not thread safe on AVR
+#10338: xbee: setting PAN ID sometimes fails
+#10341: driver_my9221 prevents further flashing of nucleo-l073rz
+#10345: frdm-k22f cannot flash after certain firmware flashed
+#10351: samd21/periph/rtt: Interrupt flags are not correctly cleared
+#10353: cpu/nrf5x_common: hwrng hangs
+
+Other issues
+------------
+
+#1263: sys: the TLSF implementation contains (a) read-before-write error(s)
+#4053: macros: RIOT_FILE_RELATIVE printing wrong file name for headers
+#4488: making the newlib thread-safe
+#5561: C++11 extensions in header files
+#5776: make: Predefining CFLAGS are parsed weirdly
+#5863: OSX+SAMR21-xpro: shell cannot handle command inputs larger than 64 chars
+#6120: Windows AVR Mega development makefile Error
+#7220: sys/fmt: Missing tests for fmt_float,fmt_lpad
+#7877: SPI connection can't read 16bit size register
+#8107: crypto/ccm: bugs in the implementation of CCM mode
+#9548: Many i2c bugs found with the i2c api rework but did not reach the release
+#9709: examples: failed assertion in dtls-echo example
+#9742: `buildtest` uses wrong build directory
+#10121: RIOT cannot compile with the latest version of macOS (10.14) and Xcode 10
+#10175: No error returned from aes_init when a key with a bad size is used
+#10206: Lua_basic example doesn't flash in b-l072z-lrwan1
+#10258: Incorrect default $PORT building for esp32-wroom-32 on macOS
+#10287: docker: docker cannot access /etc/localtime on OS X
+#10367: sam0.inc.mk: Did not find a device with serial ATML21xxxxxxxx
+
+Fixed Issues from the last release (2018.07)
+============================================
+
+#9573: pkg/lwip: assert on samr21-xpro
+#9584: drivers/cc2420: add netopt NETOPT_MAX_PACKET_SIZE
+#6437: periph/spi: Leftovers from SPI rework in #4780
+#6650: GPIO Port shift in cc2538 is unconsistent
+#6836: Issuing a reboot on ATmega328p makes the board loop-reboot
+#7846: stm32f1: I2C read functions return bogus values with unconnected devices
+#9616: at86rf2xx: can not correctly set channel after `NETOPT_STATE_RESET`
+#9548: Many i2c bugs found with the i2c api rework but did not reach the release
```

Comment:
Should this be in both known and fixed issue (I guess certain aspects were fixed and some remain?

Reply:
specifically this line: 
#9548: Many i2c bugs found with the i2c api rework but did not reach the release